frontend/src/components/modules/CreateTournament/CreateTournament.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { type SyntheticEvent, useState } from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import "./createtournament.css";
+
+const TournamentForm: React.FC = () => {
+  const today = new Date();
+
+  const [startDate, setStartDate] = useState<Date | null>(new Date());
+  const [endDate, setEndDate] = useState<Date | null>(new Date());
+
+  const [formData, setFormData] = useState({
+    tournamentName: "",
+    location: "",
+    description: "",
+    tournamentType: "",
+    organizer: false,
+    organizerEmail: "",
+    organizerTel: ""
+  });
```

Comment:
We should probably create an explicit interface for this.

Reply:
So similarly how the Matches have an interface in the backed. 

https://github.com/Koodikukkaro/Kendo-tournament-app/blob/development/backend/server/src/models/matchModel.ts#L18-L28


Having an interface for the requests makes maintaining stuff easier.

Reply:
yeah ive been working on this with using the login interface as an example. only problem is that it doesn't allow me to update the start and end date on change for some reason so still figuring out what is up with that.